backend/mr-db-sql/src/main/kotlin/io/featurehub/db/services/FeatureSqlApi.kt

```diff
@@ -211,9 +215,23 @@ class FeatureSqlApi @Inject constructor(
       existing.whoUpdated = QDbPerson().id.eq(person.person.id!!.id).findOne()
       save(existing)
       publish(existing)
+      publishChangesForMessaging(existing, lockUpdate, defaultValueUpdate, retiredUpdate, strategyUpdates)
     }
   }
 
+  private fun publishChangesForMessaging(
+    featureValue: DbFeatureValue,
+    lockUpdate: SingleFeatureValueUpdate<Boolean>,
+    defaultValueUpdate: SingleFeatureValueUpdate<String>,
+    retiredUpdate: SingleFeatureValueUpdate<Boolean>,
+    strategyUpdates: MultiFeatureValueUpdate<RolloutStrategyUpdate, RolloutStrategy>
+  ) {
+    log.trace("publishing feature messaging update for {}", featureValue)
```

Comment:
could you please drop this into an executor? Its going to take quite a while and it will delay the return of the value of the user in the front end.  There is an ExecutorSupplier interface you can inject into the FeatureSqlApi and then you can ask for an executor of X thread size. I would recommend making it externally configurable but give it a size of say - 4 - allowing 4 concurrent changes to be processed at a time. Because its an interface its mockable in your tests allowing your tests to remain synchronous.